documentation/Functions.md

```diff
@@ -173,6 +179,12 @@ cobiGenJava("path/to/java/file/MyEntity.java",[1,3,5,6,8])
 | 6 | Entity infrastructure: Creates the entity main interface and edits (by a merge) the current entity to extend the newly generated classes.|
 | 8 | CRUD SpringData Repository: Generates the entity repository (that contains the CRUD operations) in the data access layer.
 
+Note: 
+
+1. User needs to execute this command manually. 
+
+2. You need to add installCobiGen() and adaptTemplatesCobiGen() functions before adding this function for CobiGen installation and adapting the Cobigen Templates respectively.
+
```

Comment:
We can additionally mention a note that the functions uses CobiGen CLI and more information about this can be found [here](https://devonfw.com/website/pages/docs/howto_Cobigen-CLI-generation.asciidoc.html)